packages/@noma/plugin-amqplib/src/index.d.ts

```diff
@@ -0,0 +1,8 @@
+import { Connection } from "amqplib";
+
+export default interface IAmqplib {
```

Comment:
I think we should have a naming convention for all the plugin interface that makes it clear that it is not the interface of the managed module `amqplib` but of the plugin. Maybe something like `AmqplibNomaPluginResult` for the return values?


Also, I have not done a lot of typescript lately, but when I did, I remember that the official guidance said to not prefix interfaces with `I`. What do you think?

Reply:
Sure that makes sense 😄 Will rename and push.

What's your thoughts on having source files as Typescript files, and then transpiling for NPM or do you want to keep source files on github strictly in Javascript?

Reply:
Great! 

I would prefer to keep source files JS for now, but I would be open to explore the opportunity later on. For now I would like to focus on releasing version 1.0.0.